src/main.c

```diff
@@ -33,6 +33,56 @@ static void init_default_values()
 }
 
 static char *submitter = NULL;
+#define LOG_NAME_PATH_DEFAULT "D:\\kernel_tests"
+#define LOG_FILE_EXT ".log"
+static const char* name_log_format = LOG_NAME_PATH_DEFAULT "-%s" LOG_FILE_EXT;
+static char* name_log = NULL;
+
+void load_name_file(const char* file_path)
+{
+    HANDLE handle = CreateFile(
+        file_path,
+        GENERIC_READ,
+        FILE_SHARE_READ,
+        0,
+        OPEN_EXISTING,
+        FILE_ATTRIBUTE_NORMAL,
+        NULL
+    );
+    if (handle == INVALID_HANDLE_VALUE) {
+        return;
+    }
+
+    DWORD file_size = GetFileSize(handle, NULL);
```

Comment:
Since only the first line will be read from this file, it could be wise to safeguard against too large file sizes here (imagine the input is a multi-GiB file...)